docs/data/material/components/text-fields/text-fields.md

```diff
@@ -224,20 +224,16 @@ Make sure that the input is larger than the label to display correctly.
 
 ### type="number"
 
-Inputs of type="number" have potential usability issues:
+We do not recommend using type="number" with a Text Field due to potential usability issues:
```

Comment:
I think this whole section, from "We do not recommend..." to the GOV.UK link, should be wrapped in a `:::warning` callout to help draw attention to the problem.